src/App.css

```diff
@@ -28,6 +28,11 @@
   color: #61dafb;
 }
 
+a {
```

Comment:
Add this in index.css file since all the styling code is there

Reply:
done! created a new pr for this